SherpaFrontEnd/Core/Layout/MainLayout.razor.css

```diff
@@ -4,10 +4,6 @@
     flex-direction: column;
 }
 
-main {
```

Comment:
Is this something we want to do? Down the road, is this going to be easier to maintain? Maybe we should have a group discussion about it

Reply:
Can you go into a bit more detail about what the is the problem you see?

Reply:
Are we going to keep on using Tailwind for inline css or are we going for the approach of having a css file for each component, as we had mentioned?